share/eds/store.go

```diff
@@ -28,13 +33,17 @@ const (
 // Blockstore interface implementation to achieve access. The main use-case is randomized sampling
 // over the whole chain of EDS block data and getting data by namespace.
 type Store struct {
+	ctx    context.Context
+	cancel context.CancelFunc
+
 	dgstr  *dagstore.DAGStore
 	mounts *mount.Registry
 
 	topIdx index.Inverted
 	carIdx index.FullIndexRepo
 
-	basepath string
+	basepath     string
+	lastGCResult *dagstore.GCResult
```

Comment:
Ok, now I see the what was you question about testing with logs

Reply:
Add a comment that is used in tests only

Reply:
done